package.json

```diff
@@ -33,7 +34,7 @@
     "icon": "resources/icon.png"
   },
   "scripts": {
-    "test": "istanbul cover _mocha -- --exit && npm run lint",
+    "test": "istanbul cover node_modules/mocha/bin/_mocha -- --exit && npm run lint",
```

Comment:
I assume this is required for the mac testing

Reply:
You are correct. I am not sure if this works with Mac or Linux build environments. I will give it a test run in a fresh Linux installation tonight.